config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-periodics.yaml

```diff
@@ -255,11 +203,11 @@ periodics:
             # during the tests more like 3-20m is used
             cpu: 2000m
   annotations:
-    testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-aws
-    testgrid-tab-name: capa-conformance-stable
+    testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-aws, sig-release-master-informing
+    testgrid-tab-name: capa-conformance-v1alpha2-k8s-master
     testgrid-num-columns-recent: '20'
     testgrid-alert-email: kubernetes-sig-cluster-lifecycle-cluster-api-alerts@googlegroups.com
-- name: ci-cluster-api-provider-aws-make-conformance-stable-k8s-ci-artifacts
+- name: ci-cluster-api-provider-aws-make-conformance-v1alpha3-k8s-ci-artifacts
```

Comment:
Added a bash-based conformance test for v1alpha3/k8s ci artifacts. I fully expect this to fail at first, but might as well enable it now and we can work out the bugs later.